app/controllers/api/v1/lists_controller.rb

```diff
@@ -42,6 +42,6 @@ def set_list
   end
 
   def list_params
-    params.permit(:title, :replies_policy)
+    params.permit(:title, :replies_policy, :is_exclusive)
```

Comment:
I think it should just be `exclusive`, there is no precedent of `is_` prefixes in our API, all boolean arguments have the form `bot` instead of `is_bot` and so on.